src/app/ReadClient.h

```diff
@@ -645,6 +639,7 @@ class ReadClient : public Messaging::ExchangeDelegate
     uint16_t mMaxInterval              = 0;
     SubscriptionId mSubscriptionId     = 0;
     ScopedNodeId mPeer;
+    NodeId mLocal;
```

Comment:
Why are we bloating these things?  The caller can just get the local node ID from the fabric table, no?

Reply:
agree, removed, just obtain local node id from fabric table. thanks